generators/python/sdk/versions.yml

```diff
@@ -1,5 +1,13 @@
 # yaml-language-server: $schema=../../../fern-versions-yml.schema.json
 # For unreleased changes, use unreleased.yml
+- version: 4.22.1-rc0
+  changelogEntry:
+    - summary: |
+        Lazily validate model serialization.
```

Comment:
@jsklan can we add some more details about the impact this may have around memory footprint related to 2.11.0 ? 

Basically things you want to cover: 
- Pydantic 2.11 increased memory footprint
- We now defer model serialization
- memory footprint is a lot less